themes/default/content/blog/the-present-and-future-of-ai-and-iac/index.md

```diff
@@ -0,0 +1,111 @@
+
+![Pinecone reference architecture image](reference-architecture.png)
+
+The cloud-native foundations of the modern AI application wave also lead to a lot of adoption and usage of Kubernetes as an orchestration platform. Infrastructure as code with Pulumi helps teams manage the Kubernetes cluster, the workloads, and the external AI service dependencies all in a consistent way, using a unified programming model and tools, and ensuring consistent delivery of changes, the ability to easily spin up test and development environments, and consistent application of policy and insights.
+
+One particular trend in rapid growth of IaC is in fast-growing companies building the new managed AI infrastructure services themselves - from data services (e.g. [Pinecone](https://www.pinecone.io/)) to compute (e.g. [Modal](https://modal.com/)). These products face the need to manage complex cloud infrastructure as part of both multi-tenant and single-tenant solutions in order to offer simple abstracted data and compute services for their AI-application developer customers. Automation of infrastructure provisioning and changes via tools like Pulumi Automation API and Pulumi Deployments has seen rapid adoption and scaling among these customers. And the ability to deploy across many cloud providers to offer low latency data and compute services to customers, data sovereignty, and to access scarce GPU and accelerated computing resources across all cloud providers with a single consistent programming model and deployment strategy proves more important than ever.
+
+### Near Future
+
+The growth in development of AI-based applications is still in its early days, and all the trends we’re seeing are only going to accelerate over the coming year.  
+
+As these AI applications transition from prototypes to production deployment, their infrastructure needs are growing and maturing. Simple prototyping environments like Jupyter notebooks or Vercel deployments will transition into AWS/Azure/Google Cloud for the security, cost, compliance and management benefits. Larger development teams and faster customer-facing iteration will require standing up testing and ephemeral development environments, and some of the managed service dependencies will be brought in house into self-managed open source model-service, training infrastructure and data pipelines.  
+
+As patterns continue to solidify around common architectures, higher-level infrastructure components will abstract away some of this production infrastructure complexity, initially within organizations, and then in public components shared in the Pulumi Registry and elsewhere.
+
+And of course, the set of managed cloud services to support the AI ecosystem will continue to grow, expand and mature. And whether delivered as part of a major cloud platform or via a fast-growing startup, it will all be available to use productively at scale via infrastructure as code in Pulumi.
```

Comment:
```suggestion
And of course, the set of managed cloud services to support the AI ecosystem will continue to grow, expand and mature. Whether delivered as part of a major cloud platform or via a fast-growing startup, it will all be available to use productively at scale via infrastructure as code in Pulumi.
```